packages/lesswrong/server/analytics/hybridViews.ts

```diff
@@ -303,6 +312,16 @@ export function getHybridView(identifier: string): HybridView | undefined {
 
 export async function refreshHybridViews() {
   Object.values(hybridViews).map((hybridView) => void hybridView.refreshMaterializedView());
+
+  const analyticsDb = getAnalyticsConnection();
+
+  if (!analyticsDb) {
+    // eslint-disable-next-line no-console
+    console.log("No analytics DB configured, not performing VACUUM ANALYZE");
+    return;
+  }
+
+  void analyticsDb.none("VACUUM ANALYZE;");
```

Comment:
I thought running this might fix some index issues, now I think it might have caused some new ones. On balance I would guess it probably helps more than it hurts though

Reply:
Update: I'm now back to thinking this probably will help and that the issue the other day was something else (see [here](https://cea-core.slack.com/archives/C03G0AQ925P/p1692397071635369?thread_ts=1692382386.355499&cid=C03G0AQ925P), although it's not that enlightening. I really don't understand why it was grinding to a halt the other day)